hedera-node/src/test/java/com/hedera/services/txns/ethereum/EthTxDataTest.java

```diff
@@ -176,18 +180,66 @@ void whiteBoxDecodingErrors() {
 				oneByte, oneByte, oneByte, oneByte,
 				oneByte, oneByte, oneByte, oneByte,
 				oneByte);
+		var size_1 = List.of(
+				oneByte);
 
 		// legacy TX with too many RLP entries
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeAsList(size_13)));
+		// legacy TX with too few RLP entries
+		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeAsList(size_1)));
 		// type 2 TX with too many RLP entries
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 2 }, size_13)));
+		// type 2 TX with too few RLP entries
+		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 2 }, size_1)));
 		// Unsupported Transaciton Type
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 127 }, size_13)));
+		// Trimmed End Bytes
+		assertThrows(IllegalArgumentException.class, () ->
+				EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(Hex.decode(RAW_TX_TYPE_0_TRIMMED_LAST_BYTES))));
```

Comment:
```suggestion
		assertNull(EthTxData.populateEthTxData(Hex.decode(RAW_TX_TYPE_0_TRIMMED_LAST_BYTES)));
```

This should be coupled with a change to EthTxData.populateEthTxData to wrap the whole body in a 
```java
    try {
      // whole contents of populateEthTxData
    } catch(IllegalArgumentException iae) {
      return null;
    }
```
This is because headlong throws IllegalArgumentExceptions on a parsing error, and we want to return null indicating there was no valid RLP data.